_posts/2017-09-28-our-engineering-culture-methodologies-and-philosophies.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title:  "Our set of engineering culture and values"
+date:   2016-09-30 14:12:09 -0400
+categories: engineering culture
+---
+
+Every engineering group is different. Every engineering manager values different
+aspects of its team. This list was put together by the engineering team at
```

Comment:
`its` should be his, her or their (if you want to be truly inclusive for gender non-conformists)